### PR TITLE
MAINT: Fix typos

### PR DESCRIPTION
**Pull request by bwignall**
All changes should be non-semantic.

### CODE REVIEW

File: tools/refguide_check.py [hunk 1]
```diff
@@ -935,7 +935,7 @@ def main(argv):
                         help="Enforce warning checking for doctests")
     parser.add_argument("--rst", nargs='?', const='doc', default=None,
                         help=("Run also examples from *rst files"
```
Comment (eric-wieser) [inline tools/refguide_check.py]:
```suggestion
                        help=("Run also examples from *rst files "
```

**Review by eric-wieser — APPROVED** [review]

File: tools/refguide_check.py (hunk 1 above)
Comment (charris) [reply tools/refguide_check.py]: @godber Suggestion looks like a needed fix.

**Comment by bwignall** [comment]
I will push the suggested fix momentarily. Thank you, @eric-wieser and @charris .

**Comment by charris** [comment]
Thanks @bwignall .

**Pull request merged**